src/woo_publications/api/urls.py

```diff
@@ -0,0 +1,34 @@
+from django.urls import include, path, re_path
+from django.views.generic import RedirectView
+
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularJSONAPIView,
+    SpectacularRedocView,
+)
+from rest_framework import routers
+
+router = routers.DefaultRouter()
+
+
+urlpatterns = [
+    path("docs/", RedirectView.as_view(pattern_name="api-docs")),
+    re_path(
+        "v1",
+        include(
+            [
+                path(
+                    "",
+                    SpectacularJSONAPIView.as_view(schema=None),
+                    name="api-schema-json",
+                ),
+                path(
+                    "docs/",
+                    SpectacularRedocView.as_view(url_name="api-schema-json"),
+                    name="api-docs",
+                ),
+                path("schema", SpectacularAPIView.as_view(schema=None), name="schema"),
```

Comment:
Can you try if things still work when you remove this line? I think it's a legacy thing.

```suggestion
```